lib/appscale.py

```diff
@@ -54,31 +54,59 @@ class AppScale():
   APPSCALE_DIRECTORY = os.path.expanduser("~") + os.sep + ".appscale" + os.sep
 
 
+  # This is the command to terminate a deployment.
   TERMINATE = "ruby /root/appscale/AppController/terminate.rb clean"
 
 
+  # This is the commands used to upgrade a node: we need first to get the
+  # newest boostrap.sh, then invoke it.
+  GET_BOOTSTRAP = "wget -q -O /tmp/bootstrap.sh http://bootstrap.appscale.com"
+  UPGRADE = "sh /tmp/bootstrap.sh --tag last"
+
   # The usage that should be displayed to users if they call 'appscale'
   # with a bad directive or ask for help.
   USAGE = """Usage: appscale command [<args>]
 
 Available commands:
-  init: Writes a new configuration file for starting AppScale.
-  up: Starts a new AppScale instance.
-  ssh: Logs into a virtual machine in a currently running AppScale deployment.
-  status: Reports on the state of a currently running AppScale deployment.
-  deploy: Deploys a Google App Engine app to AppScale.
-  undeploy: Removes a Google App Engine app from AppScale.
-  remove: An alias for 'undeploy'.
-  get: Gets all AppController properties matching the provided regex.
-  set: Sets an AppController property to the provided value.
-  tail: Follows the output of log files in a currently running AppScale deployment.
-  logs: Collects the logs produced by an AppScale deployment.
-  relocate: Moves a hosted app to different HTTP and HTTPS ports.
-  destroy: Gracefully terminates the currently running AppScale deployment.
-  down: An alias for 'destroy'.
-  clean: Forcefully terminates all services in a cluster AppScale deployment.
-  register: Registers an AppScale deployment with the AppScale Portal.
-  help: Displays this message.
+  clean                             Forcefully terminates the AppScale
+                                    deployment and delete all data. ALL
+                                    DATA WILL BE DELETED.
+  deploy <app>                      Deploys a Google App Engine app to AppScale:
+                                    <app> can be a directory with the source
```

Comment:
Change this description to something like: "... can be the top level directory of the app or a tar.gz of the code."
"Source code" might be confusing for Java apps.